src/api.py

```
@@ -656,11 +699,197 @@ async def process_evaluation_background(evaluation_id: int):
                 )
             )
 
+async def process_evaluation_run_background(run_id: int):
+    """Background task to process an evaluation run (A/B test)"""
+    from .database import async_session
```

Ruff (0.11.9)

Redefinition of unused `async_session` from line 14

Remove definition: `async_session`

(F811)